### ISSUE TITLE
Eraser tool

### ISSUE THREAD

**Issue by KarolDaniluk** [AnnoDesigner/anno-designer#293]
Hello
Could you please add eraser tool to AD?
It would be very helpful especially to remove road tiles between the buildings.
Thanks

**Comment by FroggieFrog** [comment]
What do you mean by `eraser tool`? Would it be a shortcut?

**Do you already know you can `right click` on any tile to delete it?**

**Comment by StingMcRay** [comment]
*Or selecting the objects you want to delete, and press `[Del]`/`[Delete]`-key

**Comment by KarolDaniluk** [comment]
I know i can do that, but that requires a lot of clicking. The eraser tool works as click and drag - deleting everything that it touches. Just like eraser in paint.

**Comment by FroggieFrog** [comment]
So something like a "right click drag" function would be sufficient?

**Comment by StingMcRay** [comment]
*The Broom* method.... 

@FroggieFrog If that is possible to make that then we should change the Mouse Arrow icon too, to the Broom Icon.

**Comment by KarolDaniluk** [comment]
Yup, functionality would be the same. I thought however about broom or eraser icon somewehere on the toolbar and activated by left click and hold. That would be easier to find and understand by new users. Personally I didn't know that right-click deletes objects until you told me.

**Comment by FroggieFrog** [comment]
Yes it is sometimes not so easy to find all possible functions. There is already an issue for that (#169).

You can also check the settings menu. There are some Hotkeys (that also can be changed).
